src/System.Net.Http/src/System/Net/Http/MultipartContent.cs

```diff
@@ -34,19 +34,26 @@ public class MultipartContent : HttpContent, IEnumerable<HttpContent>
         #region Construction
 
         public MultipartContent()
-            : this("mixed", GetDefaultBoundary())
+            : this("mixed", GetDefaultBoundary(), "\r\n")
         { }
 
         public MultipartContent(string subtype)
-            : this(subtype, GetDefaultBoundary())
+            : this(subtype, GetDefaultBoundary(), "\r\n")
         { }
-
+        
         public MultipartContent(string subtype, string boundary)
+            : this(subtype, GetDefaultBoundary, "\r\n")
+        { }
+        
+        public MultipartContent(string subtype, string boundary, string crlf)
```

Comment:
>crlf

Bad name for parameter. Just ask yourself why you add this parameter. The answer is because you want to pass somthing other than CRLF.